lectures/_config.yml

```diff
@@ -32,6 +30,8 @@ sphinx:
       og_logo_url: https://assets.quantecon.org/img/qe-og-logo.png
       description: This website presents a set of lectures on python programming for economics, designed and written by Thomas J. Sargent and John Stachurski.
       keywords: Python, QuantEcon, Quantitative Economics, Economics, Sloan, Alfred P. Sloan Foundation, Tom J. Sargent, John Stachurski
+      analytics:
```

Comment:
hey @AakashGfude is this specific to `quantecon-book-theme` as `sphinx-book-theme` moved it to the top level?

Reply:
@mmcky it is because of pydata-sphinx-theme. From what I read in their latest code: https://github.com/pydata/pydata-sphinx-theme/blob/main/docs/user_guide/analytics.rst . Did `sbt` do that in their code? 
Locally, the ga code is getting included when I do this.

Reply:
interesting -- ok thanks @AakashGfude